Source/Core/Core/HW/SI/SI_DeviceGBA.cpp

```diff
@@ -258,12 +261,12 @@ int GBASockServer::Receive(u8* si_buffer, u8 bytes)
   if (!m_client)
     return 0;
 
+  sf::SocketSelector selector;
+  selector.add(*m_client);
   if (m_booted)
```

Comment:
I wonder if its worth documenting why `m_booted` affects how long we can/should wait for a socket, and if those numbers have any significance (vs. being guesses that just work in practise)

Reply:
Apparently the number didn't even work on Windows, so I replaced it with redefining what `m_booted` represents.